From 5557b9ae11b0ac373562a5fd2b3d2f8d54b871cb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Fri, 16 Nov 2018 14:04:51 -0800
Subject: [PATCH 1/2] Add runtime dependency on an rmw_implementation.

Any one of the rmw implementations can satisfy the runtime depenency.
rmw_cyclonedds is ommitted until a release is available in foxy.
---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 6d7b65c..75f8825 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-foxy-rmw-fastrtps-cpp | ros-foxy-rmw-connext-cpp
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.25.1

